CHANGELOG

```diff
@@ -1,3 +1,7 @@
+* Fri Jun 21 2019 Michael Morrone  <michael.morrone@onyxpoint.com> - 0.1.0
```

Comment:
Please include this with the previous entry since they're both the same release number with the same author. Just use the latest date.

Reply:
Done